src/Arcus.Scripting.Sql/Scripts/Invoke-AzSqlDatabaseMigration.ps1

```diff
@@ -1,25 +1,36 @@
 param(
     [Parameter(Mandatory=$true)][string] $ServerName = $(throw "Please provide the name of the SQL Server that hosts the SQL Database. (Do not include 'database.windows.net'"),
     [Parameter(Mandatory=$true)][string] $DatabaseName = $(throw "Please provide the name of the SQL Database"),
-    [Parameter(Mandatory=$true)][string] $UserName = $(throw "Please provide the UserName of the SQL Database"),
-    [Parameter(Mandatory=$true)][string] $Password = $(throw "Please provide the Password of the SQL Database"),
+    [Parameter(Mandatory=$true)][string] $UserName = $(throw "Please provide the user name of the user that must be used to perform the update"),
+    [Parameter(Mandatory=$true)][string] $Password = $(throw "Please provide the password of the user that must be used to perform the update"),
     [Parameter(Mandatory=$false)][string] $ScriptsFolder = "$PSScriptRoot/sqlScripts",
     [Parameter(Mandatory=$false)][string] $ScriptsFileFilter = "*.sql",
     [Parameter(Mandatory=$false)][string] $DatabaseSchema = "dbo"
 )
 
+#Import needed for Azure Powershell Release Step
+Import-Module SqlServer
```

Comment:
Why is this here again? We can added it to the `RequiredModules` in the `.psd1` file, but it should not be here I suppose.

Reply:
Removed it.